pages/common/csvcut.md

```diff
@@ -0,0 +1,20 @@
+# csvcut
+
+> Filter and truncate CSV files. Like unix "cut" command, but for tabular data.
```

Comment:
Capitalize "Unix", and either write "Unix's cut command", or "the Unix cut command" (I prefer the former). Also, it would probably be more appropriate to format "cut" with backticks, rather than quote marks.